openff/toolkit/tests/test_molecule.py

```diff
@@ -3405,21 +3405,18 @@ def test_assign_partial_charges(self, toolkit, method):
         # Check total charge
         charges_sum_unitless = initial_charges.sum().m_as(unit.elementary_charge)
         total_charge_unitless = molecule.total_charge.m_as(unit.elementary_charge)
-        # if abs(charges_sum_unitless - total_charge_unitless) > 0.0001:
-        # print(
-        #     "molecule {}    charge_sum {}     molecule.total_charge {}".format(
-        #         molecule.name, charges_sum_unitless, total_charge_unitless
-        #     )
-        # )
-        np.allclose(charges_sum_unitless, total_charge_unitless, atol=0.002)
+
+        np.testing.assert_allclose(
+            charges_sum_unitless, total_charge_unitless, atol=0.002
+        )
```

Comment:
I actually kinda wonder if this line was doing anything (as it was missing an `assert`)